.github/workflows/run-tests.yml

```diff
@@ -27,9 +27,19 @@ jobs:
           path: ~/.npm
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: ${{ runner.os }}-node-
+      - name: Security audit
+        run: npm audit    
       - name: Install dependencies
         run: npm ci
       - name: Run lint
         run: npm run lint
       - name: Run unit tests
         run: npm test
+
+  release:
```

Comment:
Not asking for a change here, just for us to think about whether we should separate out the pre and post-merge workflows. If you look in github actions for something with does both but uses a single workflow it's really had to tell which runs were to create a release, and which were just on PRs.

I _think_ (but am not sure) we might have to do it anyway to use the concurrency feature 🤔 

Anyway I don't think you should change it here, just something to ponder on when we start doing the others and rely on GHA for releases

Reply:
I'm leaning towards splitting it out.

Reply:
@jfharden Can you check if this [commit](https://github.com/alphagov/pay-stream-s3-sqs/pull/89/commits/446ef49cec24e5187010df4872284154ed72e611) please?